unreleased/redisreceiver-version.yaml

```diff
@@ -0,0 +1,15 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Is this really a breaking change? I would have expected this to be an enhancement as it's additive.

Reply:
It depends on how we define breaking changes for emitted data. I would say it is a breaking change. It introduces new attribute which can be unexpected on the metric backends and generated new MTS's

Reply:
So as @dmitryax described, this attributed will be addd to all emitted metrics as an additional attribute/label. If consumer has strict attributes check, their workflows will be broken.

Reply:
I'm ok with moving it to enhancement given that it's not a significantly breaking for the emitted data

Reply:
i would expect a changing an existing attribute to be breaking, but not adding a new one. 

> Adding of new metrics, spans, span events or log records and adding of new attributes to spans, span events, log records or resources are considered additive, non-breaking changes and are always allowed for Unstable and Stable instrumentations.

https://github.com/open-telemetry/opentelemetry-specification/blob/cba6938951a789f861303d1b4ef3634ef00b9aa5/specification/telemetry-stability.md

Reply:
Ok let's move it to enhancement

Reply:
Updated.